roles/openshift_facts/tasks/main.yml

```diff
@@ -1,11 +1,9 @@
 ---
-- name: Verify Ansible version is greater than 1.8.0 and not 1.9.0 and not 1.9.0.1
-  assert:
-    that:
-    - ansible_version | version_compare('1.8.0', 'ge')
-    - ansible_version | version_compare('1.9.0', 'ne')
-    - ansible_version | version_compare('1.9.0.1', 'ne')
-    
+- name: Verify Ansible version is greater than or equal to 1.9.4 and less than 2.0
+  fail:
+    msg: "Unsupported ansible version: {{ ansible_version }} found"
+    when: ansible_version | version_compare('1.9.4', 'lt') or ansible_version | version_compare('2.0', 'ge')
```

Comment:
`when:` is indented two extra spaces also I think we could use `ansible_version.full`

```
failed: [10.8.52.230] => {"failed": true}
msg: Unsupported ansible version: {'major': 1, 'full': '1.9.4', 'string': '1.9.4\n  configured module search path = None', 'minor': 9, 'revision': 4} found
```